capstone-api/models/User.js

```diff
@@ -230,17 +221,101 @@ class User {
   static async getTopGenres(username) {
     const query = new Parse.Query('Preferences');
     query.equalTo("username", username);
-    let result = await query.first({}); 
-    return result.get('topGenres');
+    return (await query.first({})).get('topGenres');
   }
 
   static async getTopArtists(username) {
     const query = new Parse.Query('Preferences');
     query.equalTo("username", username);
+    return (await query.first({})).get('topArtists');
+  }
+
+  static async createGroup(groupInfo) {
+    let {username, groupName, description, isPrivate, genres, isAdmin } = groupInfo;
+    let group = new Parse.Object('Group');
+
+    group.set("name", groupName);
+    group.set('description', description);
+    group.set('isPrivate', isPrivate);
+    group.set('genres', genres);
+
+    let relationship = new Parse.Object('UserGroup');
+    relationship.set("username", username);
+    relationship.set('groupName', groupName);
+    relationship.set('isAdmin', isAdmin);
+
+    try {
+        await group.save();
+        await relationship.save();
+        return true;
+    } catch (error) {
+        return error.message;
```

Comment:
Is there a more elegant way to handle the message rather than returning the message? Maybe adding some sort of UI to indicate what the error is or including information that might be useful for debugging, such as what group name or user you are trying to add?